### PR TITLE
fix:pyee latest version

### PR DESCRIPTION
**Pull request by JarbasAl**
## Summary by CodeRabbit

- **Chores**
	- Updated the import statement for the `ExecutorEventEmitter` class to reflect its new source.
	- Revised the version constraint for the `pyee` package in `requirements.txt`, increasing the minimum required version.
  
- **Tests**
	- Adjusted the import path for `ExecutorEventEmitter` in unit tests to align with the updated module structure.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ovos-config>=0.0.12,<1.0.0
 ovos-utils>=0.0.38,<1.0.0
 websocket-client>=0.54.0
-pyee>=8.1.0, < 9.0.0
+pyee>=12.0.0
```
Comment (mikejgray) [inline requirements.txt:4]: I think this is backwards compatible, probably don't need to force a newer version

Comment (JarbasAl) [reply requirements.txt:4]: it's not, [old import no longer available](https://github.com/jfhbrook/pyee/blob/main/pyee/__init__.py)

## 2024/08/30 Version 12.0.0

- Remove deprecated imports:
  - `pyee.BaseEventEmitter`
  - `pyee.AsyncIOEventEmitter`
  - `pyee.TwistedEventEmitter`
  - `pyee.ExecutorEventEmitter`
  - `pyee.TrioEventEmitter`
- Add `PyeeError` which inherits from `PyeeException`, and use throughout
- Deprecate direct use of `PyeeException`

Comment (JarbasAl) [reply requirements.txt:4]: he, seems the imports are completely gone, not even just moved around...

**Comment by JarbasAl** [comment]
not sure why tests are failing and no time to bug hunt, so i'll leave this on hold for now unless someone wants to take over, expected a quick fix but instead...

```
ovos_bus_client/client/client.py:10: in <module>
    from pyee.executor import ExecutorEventEmitter
E   ModuleNotFoundError: No module named 'pyee.executor'
```

File: requirements.txt (hunk 1 above)
Comment (JarbasAl) [reply requirements.txt:4]: https://github.com/OpenVoiceOS/ovos-bus-client/pull/124 with backwards compat we can only go up to 11.1.1

**Pull request closed without merging**